src/xcode/ENA/ENA/Source/Scenes/Settings/NotificationSettingsViewController.swift

```diff
@@ -87,12 +90,10 @@ class NotificationSettingsViewController: UIViewController {
 	private func notificationSettings() {
 		let center = UNUserNotificationCenter.current()
 
-		center.getNotificationSettings { [weak self] settings in
+		center.requestAuthorization(options: [.alert, .sound, .badge]) { [weak self] granted, error in
```

Comment:
If we don't want to handle the error here, please remove unused parameter (_). It produces a warning. But, I would rather use it and log the error here, the same way as it was logged before in ```SettingsViewController```.

Reply:
Fixed